src/site/content/en/blog/next-gen-css-2019/index.md

```diff
@@ -164,7 +163,7 @@ Here are the new queries we think developers will be most excited about:
 *   [prefers-reduced-transparency](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-transparency)
 *   [forced-colors](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/forced-colors)
 *   [inverted-colors](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/inverted-colors)
-*   [light-level](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/light-level)
+*   [light-level](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/Media_features)
```

Comment:
This link is not working either, light-level dcs seem to be MIA.  I found this https://reference.codeproject.com/Book/css/@media/light-level but couldn't find the reference in CSS spec https://drafts.csswg.org/mediaqueries-4/#light-level. @una @argyleink halp pls :)

Reply:
It was part of the level 5 spec when they wrote this article. It has [since been removed](https://drafts.csswg.org/mediaqueries-5/#changes-since-2020-07-15). Fortunately this feature isn't reference in text, so I've just removed it.